.github/workflows/renovate-deploy.yml

```diff
@@ -22,14 +22,20 @@ jobs:
           repositories=$(echo "${{ github.event.inputs.deployRepository }}" | cut -d '/' -f2)
           echo "owner=$owner" >> "$GITHUB_OUTPUT"
           echo "repositories=$repositories" >> "$GITHUB_OUTPUT"
+      - name: Load Secrets from Vault
+        if: steps.override.outputs.use-gh-app == 'true'
```

Comment:
We can remove this `if` check, because this action will only run from this repo, it doesn't run from PRs (or forks).

Reply:
```suggestion
```